### ISSUE TITLE
Not working with Zoom

### ISSUE THREAD

**Issue by MandaNyAina** [kyleneideck/BackgroundMusic#650]
Description of the bug

It doesn't mute ZOOM meeting, unless I mute all system sounds (which I obviously don't want to).

Steps to reproduce

Launch ZOOM meeting
Launch Background Music
Mute everything BUT the system sounds
See that ZOOM sound is still on

**Comment by cron410** [comment]
Have you tried with BackgroundMusic already running before the meeting starts?

**Comment by MandaNyAina** [comment]
> Have you tried with BackgroundMusic already running before the meeting starts?

Yes, I tried to uninstall zoom and re-install but nothing change

**Comment by cron410** [comment]
Try the snapshot. 0.4.0-SNAPSHOT-b38f6dd

**Comment by Maks-Chornyi** [comment]
Can proove that when i chose 'background music'  in the ZOOM settings (for speaker) i got possibility to increase/decrease Zoom's sound level.

**Comment by bgiromini** [comment]
I just tried in a zoom meeting and as soon I as changing the Speaker to Background Music, the audio gets robotic and garbled.
